### ISSUE TITLE
Can't find path to dtutil in registry

### ISSUE THREAD

**Issue by jennaallen** [BIDeveloperExtensions/bideveloperextensions#81]
I keep getting this error when trying to deploy an SSIS package to the server:
Error : BIDS Helper encountered an error when deploying packages:
Can't find path to dtutil in registry! Please make sure you have the SSIS service installed from the SQLServer2017 install media
  at BIDSHelper.SSIS.DeployPackagesPlugin.DeployProject(Project proj, IOutputWindow outputWindow, Array selectedItems, Boolean bCreateBat) in C:\projects\BIDS Helper\GitHub\bideveloperextensions\SSIS\DeployPackagesPlugin.cs:line 301

and this
![image](https://user-images.githubusercontent.com/17113931/67061542-7f11f000-f115-11e9-898e-5418f9e09f94.png)

**Version**
![image](https://user-images.githubusercontent.com/17113931/67061733-08292700-f116-11e9-9635-45002a7f58fa.png)

**To Reproduce**
1. I have a project that uses the package deployment model
2. I right click on the project and hit 'Deploy'
3. See error

I uninstalled VS 2017 and SSDT and followed the steps here for re-installing https://github.com/BIDeveloperExtensions/bideveloperextensions/issues/59#issuecomment-484666551 but I still get the error.

Thanks in advance for any guidance.

**Comment by furmangg** [comment]
The first error means you need to run the SQL Server 2017 installer and install Integration Services. It can be Developer Edition. 

The second error is something we should fix. For now it sounds like you may be able to workaround it by docking the Output window. Let me know if that works.

**Comment by jennaallen** [comment]
Yes, this worked! Thank you for the quick response and the super helpful extension. 
As you predicted, when the Output window was docked, I didn't get the second error message.
